src/Files.App/UserControls/OpacityIcon.xaml.cs

```diff
@@ -0,0 +1,39 @@
+using CommunityToolkit.WinUI.UI;
+using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Controls;
+
+// The User Control item template is documented at https://go.microsoft.com/fwlink/?LinkId=234236
+
+namespace Files.App.UserControls
+{
+	public sealed partial class OpacityIcon : ContentControl
```

Comment:
Do we need to inherit from Content control? I think a "regular" templated control would be better for performance.

Reply:
`Control`?

Reply:
Yes

Reply:
I switched it to `Control`.